### DISCUSSION TITLE
Export导出HDFS 失效以及导出查询结果集并发无法启用

### DISCUSSION THREAD

**Discussion by fdf1779** [apache/doris#29450] [A - General / Q&A]
目前使用的Doirs版本为2.0.3
我现在有一个场景需要将查询的结果数据导出到HDFS，同时要保证导出效率尽可能的高。
使用Export时，export任务启动之后没有文件导出，并且导出状态持续保持在exporting 0%，查看日志有警告信息
ReportExecStatus() runtime error, query 97c8d582d0074e49-8b0515412e476f07 with type SELECT does not exist

于是我想使用导出查询结果集的方式，导出OLAP表 random 分桶的查询结果集，在设置set enable_parallel_outfile = true;  
对结果进行where 过滤后依旧无法启用并发导出
还有别的方法可以快速的将结果集导出到HDFS吗

**Comment by LemonLiTree** [comment]
可能需要分析下日志了。
